clang/lib/Serialization/ASTReader.cpp

```diff
@@ -2616,6 +2616,18 @@ bool ASTReader::shouldDisableValidationForFile(
   return false;
 }
 
+namespace {
+
+std::pair<StringRef, StringRef>
+getUnresolvedInputFilenames(const ASTReader::RecordData &Record,
```

Comment:
Nit: Per [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#id59), this should be `static` instead of wrapped inside an anonymous namespace:
> make anonymous namespaces as small as possible, and only use them for class declarations